documents/Libraries/stdlib/genosl/mx_transformnormal3M.inline

```diff
@@ -0,0 +1 @@
+transform({{mat}}, (normal){{in}})
```

Comment:
Double checking -- this produces a mat4 * vec3 call -- correct?

Reply:
I think it's working because 3x3 matrices are converted to 4x4 ones due to the code you wrote.